SLAM/include/CameraFrame.h

```diff
@@ -0,0 +1,33 @@
+//
+// Created by Kyler Jensen on 2/15/17.
+//
+
+#ifndef SLAM_CAMERAFRAME_H
+#define SLAM_CAMERAFRAME_H
+
+#include "_tmp.h"
+#include "RGBColor.h"
+#include <vector>
+
+namespace slam {
+
+    class CameraFrame {
+
+    private:
+        Chrono timestamp;
+        std::vector<std::vector<RGBColor>> data;
+
+    public:
+        CameraFrame(unsigned long width, unsigned long height, Chrono timestamp);
+        CameraFrame(unsigned long width, unsigned long height);
+        Chrono getTimestamp();
+        RGBColor getPixel(unsigned long x, unsigned long y);
```

Comment:
I just had to comment that I really _really_ like the use of the `Chrono` and `RGBColor` types here. So clean. 😃